src/site/_includes/components/Export.js

```diff
@@ -64,6 +64,18 @@ async function Export() {
     return '';
   }
 
+  const authors = JSON.stringify(page?.template?.frontMatter?.data.authors);
+
+  const frontMatter = `project_path: /_project.yaml
+book_path: /_book.yaml
+
+{% import "/_macros.html" as macros %}
+
+# ${page?.template?.frontMatter?.data.title}
```

Comment:
Does this get the computed title? Haven't checked if there are cases where we use one.

Reply:
It does not. We don't use computed titles for posts though.